src/main/kotlin/frc/robot/subsystems/ReefCommands.kt

```diff
@@ -88,13 +89,17 @@ fun outtakeCoral(): Command =
                 ))
             .withTimeout(0.5),
         gripper.slowOuttake(true).withTimeout(0.1),
-        moveDefaultPosition().onlyIf(gripper.hasCoral.negate())
+        moveDefaultPosition().onlyIf(gripper.hasCoral.negate().and{!DriverStation.isAutonomous()})
     )
 
 // TODO: Add Coral Simulation
 
 fun moveDefaultPosition(): Command =
-    sequence(elevator.feeder(), waitUntil(elevator.atSetpoint), wrist.feeder())
+    sequence(
+            elevator.feeder(),
+            waitUntil(elevator.atSetpoint),
+            wrist.feeder()
+        )
```

Comment:
```suggestion
    elevator.run { feeder().until(atSetpoint()) }.andThen(wrist.feeder())
```
As usual, this is clearer using infix notation... 😉
```kt
elevator.run { feeder() until atSetpoint() } andThen wrist.feeder()
```
Edit: I'm not sure if that'll work because of `Subsystem.run()`. An alternative is to use `let` or `with`, but the benefit will be smaller:
```suggestion
    elevator.let { it.feeder().until(it.atSetpoint()) }.andThen(wrist.feeder())
```
```suggestion
    with(elevator) { feeder().until(atSetpoint()) }.andThen(wrist.feeder())
```